### ISSUE TITLE
panctl : ImportError: cannot import name 'HTML'

### ISSUE THREAD

**Issue by arthurlutz** [matrix-org/pantalaimon#32]
```
panctl 
Traceback (most recent call last):
  File "/home/arthur/.virtualenvs/pantalaimon/bin/panctl", line 5, in <module>
    from pantalaimon.panctl import main
  File "/home/arthur/.virtualenvs/pantalaimon/lib/python3.6/site-packages/pantalaimon/panctl.py", line 27, in <module>
    from prompt_toolkit import HTML, PromptSession, print_formatted_text
ImportError: cannot import name 'HTML'
```

**Comment by arthurlutz** [comment]
Maybe the dependency to prompt_toolkit needs to be a bit more precise ? 

I realised that I had done `toggleglobalsitepackages` in this env, so I was probably on 

```
python3-prompt-toolkit/bionic,bionic,now 1.0.15-1 all [installed,automatic]
```

**Comment by poljar** [comment]
I can't reproduce that on prompt toolkit 2 nor 3. Does that line show prompt toolkit 1.0 as the installed version?

**Comment by arthurlutz** [comment]
I think this is fixed by #34

**Comment by poljar** [comment]
Fixed now.
